src/store/index.ts

```diff
@@ -0,0 +1,72 @@
+import { createStore, createLogger } from "vuex";
+import { Dapp, Account, DappLocalState } from "@/types";
+import api from "@/api/dapps";
+
+const debug = process.env.NODE_ENV !== "production";
+
+export default createStore({
+	state: {
+		hasAlgoSigner: false,
+		dapps: [] as Dapp[],
+		userAccounts: [] as Account[],
+		activeDapps: [] as DappLocalState[],
+	},
```

Comment:
please create a Store class instead of using a dynamic object with `as` (type assertions). We should not use `as`.